components/widgets/users/avatar/index.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Ok, here is clearly not waiting for future component connections, is this the new pattern for components? always in its own folder? @calebroseland @deanwhillier 
I think the the proliferation of other files around the components like the `scss`, `stories.tsx`, `test.tsx`, `__snapshots`... probably makes a lot of sense to make it this way.

Reply:
Yes, this will be the new pattern moving forward. :)